keras_hub/src/utils/timm/convert_efficientnet.py

```diff
@@ -189,15 +236,22 @@ def port_batch_normalization(keras_layer, hf_weight_prefix):
 
     # Stages
     num_stacks = len(backbone.stackwise_kernel_sizes)
+
+    depth_coefficient = VARIANT_MAP[variant]["depth_coefficient"]
```

Comment:
if we follow the suggestion about arg renaming above, we can get rid of this whole block. Just update above from

```
"width_coefficient": 0.8,
"depth_coefficient": 0.9,
```

to

```
"stackwise_width_coefficient": [0.8] * 6,
"stackwise_depth_coefficient": [0.9] * 6,
```

Reply:
I made it so both ways should work.